docs/getting_started/notifications.md

```diff
@@ -25,4 +27,36 @@ As a result, you should see the following logs on the server side
 ```
 07:45:02.684 [info] New incoming server WebSocket connection, accepting
 07:45:02.688 [info] Server WS authenticated.
+```
+
+To receive server notifications through webhooks you have to pass `webhook_url` during creating room.
+Example using Elixir Server SDK:
+```elixir
+server_address = "localhost:5002"
+server_api_token = "development"
+webhook_url = "http://localhost:5003/"
+
+client = Jellyfish.Client.new(server_address: server_address, server_api_token: server_api_token)
+{:ok, %Jellyfish.Room{id: room_id}, jellyfish_address} = Jellyfish.Room.create(client, max_peers: 10, webhook_url: webhook_url)
+```
+
+To decode received notification through webhook, you can use a function from SDK.
+Example using Elixir Server SDK:
+```elixir
+  notification = %{
+    "notification" => <<18, 76, 10, 36, 102, 98, 102, 52, 49, 57, 48, 99, 45, 53, 99, 55, 54, 45, 52,
+    49, 53, 99, 45, 56, 57, 51, 57, 45, 53, 50, 99, 54, 101, 100, 50, 48, 56, 54,
+    56, 98, 18, 36, 99, 55, 50, 51, 54, 53, 56, 55, 45, 53, 100, 102, 56, 45, 52,
+    98, 52, 49, 45, 98, 54, 101, 52, 45, 50, 54, 56, 101, 55, 49, 49, 51, 51, 101,
+    101, 50>>
+  } 
```

Comment:
That' a little bit low level. I belive it would be better to show how we define POST route and pass the body of incoming post into our receive function